star-war-character-app/src/pages/Vehicle/VehicleDetails/VehicleDetails.tsx

```diff
@@ -0,0 +1,101 @@
+import { Link, useParams } from "react-router-dom";
+import { IRootState, useAppDispatch } from "../../../redux/store";
+import { useSelector } from "react-redux";
+import { Fragment, useEffect } from "react";
+import { getIndividualVehicleActions } from "../../../redux/VehicleSlice/VehicleAsyncThunk";
+import { Strings } from "../../../resource/Strings";
```

Comment:
Setup relative path

Reply:
I have resolved this issue in this PR https://github.com/Harsh-Upadhyay-AT/Star-War-Character-App/pull/2 Please check it.